### DISCUSSION TITLE
Ideas for beginner projects to do?

### DISCUSSION THREAD

**Discussion by ygorz** [Cyfrin/foundry-full-course-cu#2969] [General]
Hello everyone, I'm slowly making my way through the course (currently on the Foundry Smart Contract Lottery section) and had a quick question. Are there any recommendations for beginner projects to work on when we've acquired enough knowledge, just to have something to put our skills to a test and build on our own without anyones help? I'm not sure if there are suggestions made further in the course and I just haven't gotten to it. 

Following a lot has been great and I'm learning a lot but I realize at some point I'll have to actually build things on my own if I want to take my learning further and actually have things to put on a portfolio. I'm currently taking this course while holding down a full time job so admittedly my creative juices aren't really flowing with ideas and would appreciate any suggestions people had. Thank you!

**Comment by EngrPips** [comment]
Hello @ygorz, The best thing to do is create everything you created while coding along with Patrick all by yourself without any help whatsoever and that way you will see how much you have truly learnt and you will start getting ideas of your own on how to better develop the code and maybe even to add some functionality that Patrick didn't add.
